completions/ssh

```diff
@@ -440,21 +444,21 @@ _scp_remote_files()
 
     # default to home dir of specified user on remote host
     if [[ -z $path ]]; then
-        path=$(ssh -o 'Batchmode yes' "$userhost" pwd 2>/dev/null)
+        path=$(ssh $__sshoption -o 'Batchmode yes' "$userhost" pwd 2>/dev/null)
```

Comment:
Have you actually tested this code? I haven't tried it by myself, but it doesn't seem to work because `$__sshoption` (whose content is `-p NUMBER`) would appear to become a single word without word splitting as `IFS` is set to `$'\n'` in the beginning of this function.

Reply:
Hmm, OK. Somehow this works because `ssh` seems to accept the option of the form `'-p NUMBER'` (although I'm not sure if this is your original intention). I have confirmed that the following command works in my environment where 8022 is the port number of my local sshd:

```bash
$ ssh '-p 8022' localhost
```